src/plugins/ncr/screens/node/index.jsx

```diff
@@ -178,6 +182,14 @@ export default class AbstractNodeScreen extends React.Component {
     this.setState(({ isSaveDropDownOpen }) => ({ isSaveDropDownOpen: !isSaveDropDownOpen }));
   }
 
+  renderAdditionalPrimaryActions() {
```

Comment:
@cohencodes seems like this extra logic is not needed.  if this just returns `</>` by default you only need override this method and don't need renderAdditionalPrimaryActions and getAdditionalPrimaryActions